.github/scripts/test-directory-unchanged.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/sh
 set -ex
 
-cd "$(dirname "$0")/../../$1"
+# use the repo root directory as "--git-dir"
+cd "$(dirname "$0")/../.."
```

Comment:
This is important as the original `cd "$(dirname "$0")/../../$1"` did not detected any changes in case of the checked directory was containing `.git`.

Example: `git clone https://github.com/derickr/timelib.git ext/date/lib`